web/lib/js/main.js

```diff
@@ -83,8 +84,32 @@ function initialise() {
         });
     }, 1000);
 
-    setInterval(function () {
-        $.getJSON(g_api + "/commands/").done(function (data) {
+    processCommands();
```

Comment:
Starting long-poll chain here, without setTimeout (delay is enforced by long-poll itself)